From stable-bounces@linux.kernel.org Wed Nov 21 14:12:51 2007
From: Dmitry Torokhov <dmitry.torokhov@gmail.com>
Date: Wed, 21 Nov 2007 17:12:26 -0500
Subject: Input: ALPS - add signature for ThinkPad R61
To: linux-stable <stable@kernel.org>
Cc: Dmitry Torokhov <dmitry.torokhov@gmail.com>
Message-ID: <4744AD4A.7000307@redhat.com>

From: Dmitry Torokhov <dmitry.torokhov@gmail.com>

changeset f493018ebc3f94d64e12bc848db0906700bf73a2 in mainline.

Input: ALPS - add signature for ThinkPad R61

Signed-off-by: Dmitry Torokhov <dtor@mail.ru>
Cc: Chuck Ebbert <cebbert@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/input/mouse/alps.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/input/mouse/alps.c
+++ b/drivers/input/mouse/alps.c
@@ -48,6 +48,7 @@ static const struct alps_model_info alps
 	{ { 0x63, 0x02, 0x50 },	0xef, 0xef, ALPS_FW_BK_1 },		/* NEC Versa L320 */
 	{ { 0x63, 0x02, 0x64 },	0xf8, 0xf8, 0 },
 	{ { 0x63, 0x03, 0xc8 }, 0xf8, 0xf8, ALPS_PASS },		/* Dell Latitude D800 */
+	{ { 0x73, 0x00, 0x0a },	0xf8, 0xf8, ALPS_DUALPOINT },		/* ThinkPad R61 8918-5QG */
 	{ { 0x73, 0x02, 0x0a },	0xf8, 0xf8, 0 },
 	{ { 0x73, 0x02, 0x14 },	0xf8, 0xf8, ALPS_FW_BK_2 },		/* Ahtec Laptop */
 	{ { 0x20, 0x02, 0x0e },	0xf8, 0xf8, ALPS_PASS | ALPS_DUALPOINT }, /* XXX */
